docs/doc_4_authentication.md

```diff
@@ -40,3 +39,160 @@ func main() {
 	fmt.Printf("Projects: %v\n", projects)
 }
 ```
+
+## (Preview) Using the Atlas Go SDK with Service Account Authentication
+
+Atlas SDK Go provides OAuth Authentication using Service Accounts (currently available as a [Preview](https://www.mongodb.com/resources/beta-programs) feature)
+Service Account implement OAuth [client_credentials](https://oauth.net/2/grant-types/client-credentials) grant.
```

Comment:
"Service Account" is, I gather, not a collective term. Our possible alternatives are: 

```suggestion
Service Accounts implement OAuth [client_credentials](https://oauth.net/2/grant-types/client-credentials) grants.
```

OR

```suggestion
A Service Account implements an OAuth [client_credentials](https://oauth.net/2/grant-types/client-credentials) grant.
```

The latter approach makes more sense if you are going to continue with a description of features of a single Service Account, especially if that description is going to involve assumptions about one instance of a Service Account (e.g. specific values for parameters). In this case, we are aiming at a higher-level conceptual summary, so I recommend the former approach that uses plurals.